test/unit/lib/record_filter_test.rb

```diff
@@ -234,7 +234,8 @@ def test_provider_filter
     prov = Provider.default_provider
 
     patients.each do |patient|
-      patient.provider_performances = [{ provider_id: prov.id }]
+      patient.providers = nil
```

Comment:
Does this work? Wouldn't it have to be `patient.providers = []`, so the provider could get pushed onto it?

Reply:
both approaches seem to work equivalently

has_and_belongs_to_many :providers, class_name: 'CQM::Provider'

my guess is mongoid/activerecord magic with "has_and_belongs_to_many"